content/docs/guides/responsive_amp@es.md

```diff
@@ -60,13 +36,14 @@ Por ejemplo:
         background-color: grey;
       }
     </style>
-    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    ...
   </head>
 [/sourcecode]
 
-**Importante:**
+{% call callout('Important', type='caution') %}
```

Comment:
For the English word "Important", can you change it to the Spanish word, which I believe is "Importante"?